### PR TITLE
Update serverless installation docs

### PR DESCRIPTION
**Pull request by tianchu**
### What does this PR do?

Update the recently added serverless installation docs.

### Preview link

https://docs-staging.datadoghq.com/tian.chu/serverless-set-up/serverless/installation/python

Check preview base path using the URL in details in `preview` status check.

### CODE REVIEW

File: content/en/serverless/installation/nodejs.md [hunk 1]
```diff
@@ -10,40 +10,38 @@ further_reading:
       text: 'Installing Ruby Serverless Monitoring'
 ---
 
-After you have [installed the AWS integration][1], use Node.js to instrument your application to send metrics, logs, and traces to Datadog. 
+After you have [installed the AWS integration][1], choose one of the following method to instrument your application to send metrics, logs, and traces to Datadog.
```
Comment (apigirl) [inline content/en/serverless/installation/nodejs.md:13]:
```suggestion
After you have [installed the AWS integration][1], choose one of the following methods to instrument your application to send metrics, logs, and traces to Datadog.
```

File: content/en/serverless/installation/nodejs.md [hunk 2]
```diff
@@ -159,46 +110,62 @@ npm install datadog-lambda-js
...
 
-Then, using the AWS Console or the AWS CLI, add a new `DD_FLUSH_TO_LOG` environment variable set to `true`. This step needs to be repeated for every function you wish to trace.
+See the [latest release][3].
 
-### Instrument your code
+### Configure the Function
 
-Instrument your functions to ingest traces into Datadog:
+1. Set your function's handler to `/opt/nodejs/node_modules/datadog-lambda-js/handler.handler` if using the layer, or `node_modules/datadog-lambda-js/dist/handler.handler` if using the package.
+1. Set the environment variable `DD_LAMBDA_HANDLER` to your original handler, e.g., `myfunc.handler`.
```
Comment (apigirl) [inline content/en/serverless/installation/nodejs.md:118]:
```suggestion
2. Set the environment variable `DD_LAMBDA_HANDLER` to your original handler, for example, `myfunc.handler`.
```

File: content/en/serverless/installation/nodejs.md [hunk 3]
```diff
@@ -159,46 +110,62 @@ npm install datadog-lambda-js
...
+1. Set environment variable `DD_TRACE_ENABLED` to `true`.
```
Comment (apigirl) [inline content/en/serverless/installation/nodejs.md:119]:
```suggestion
3. Set the environment variable `DD_TRACE_ENABLED` to `true`.
```

File: content/en/serverless/installation/nodejs.md [hunk 4]
```diff
@@ -159,46 +110,62 @@ npm install datadog-lambda-js
...
+1. Set environment variable `DD_FLUSH_TO_LOG` to `true`.
```
Comment (apigirl) [inline content/en/serverless/installation/nodejs.md:120]:
```suggestion
4. Set the environment variable `DD_FLUSH_TO_LOG` to `true`.
```

File: content/en/serverless/installation/nodejs.md [hunk 5]
```diff
@@ -159,46 +110,62 @@ npm install datadog-lambda-js
...
+[2]: https://github.com/DataDog/datadog-lambda-layer-js/releases
+[3]: https://www.npmjs.com/package/datadog-lambda-js
+[4]: https://docs.datadoghq.com/serverless/troubleshooting/installing_the_forwarder
+[5]: https://docs.datadoghq.com/serverless/troubleshooting/installing_the_forwarder/#ddfetchlambdatags
+[6]: https://docs.datadoghq.com/integrations/amazon_web_services/?tab=automaticcloudformation#send-aws-service-logs-to-datadog
+{{% /tab %}}
+{{< /tabs >}}
 
-You can quickly verify that you’ve installed the Datadog Forwarder on the [AWS Forwarder page][1] For more information on the Forwarder ARN, or to install the forwarder see the [official CloudFormation documentation][1]. Make sure the Datadog Forwarder is in the same AWS region as the Lambda functions you are monitoring.
+## Quick Start
```
Comment (apigirl) [inline content/en/serverless/installation/nodejs.md:141]:
```suggestion
## Viewing your data
```

File: content/en/serverless/installation/nodejs.md [hunk 6]
```diff
@@ -159,46 +110,62 @@ npm install datadog-lambda-js
...
 
+After you have configured your function following the steps above, you can view the enhanced metrics, logs and traces on the [Serverless page][2]. If you would like to submit a custom metric or manually instrument a function, see the sample code below.
```
Comment (apigirl) [inline content/en/serverless/installation/nodejs.md:143]:
```suggestion
After you have configured your function, you can view the enhanced metrics, logs and traces on the [Serverless page][2]. If you would like to submit a custom metric or manually instrument a function, see the sample code below.
```

File: content/en/serverless/installation/python.md [hunk 7]
```diff
@@ -10,40 +10,38 @@ further_reading:
       text: 'Installing Ruby Serverless Monitoring'
 ---
 
-After you have [installed the AWS integration][1], use Python to instrument your application to send metrics, logs, and traces to Datadog. 
+After you have [installed the AWS integration][1], choose one of the following method to instrument your application to send metrics, logs, and traces to Datadog.
```
Comment (apigirl) [inline content/en/serverless/installation/python.md:13]:
```suggestion
After you have [installed the AWS integration][1], choose one of the following methods to instrument your application to send metrics, logs, and traces to Datadog.
```

File: content/en/serverless/installation/python.md [hunk 8]
```diff
@@ -74,11 +72,17 @@ To install the Datadog Lambda Library with the SAM macro, follow these steps:
...
 ### Install the Datadog Lambda Library
 
-The Datadog Lambda Library can be imported as a layer. Its ARN includes a region, language runtime, and version. Construct yours in the following format:
+You can either install the Datadog Lambda library as a layer or as a Python package.
+
+The minor version of the `datadog-lambda` package always matches the layer version. E.g., datadog-lambda v0.5.0 matches the content of layer version 5.
+
+#### Using the Layer
+
+[Configure the layers][1] for your Lambda function using the ARN in the following format.
```
Comment (apigirl) [inline content/en/serverless/installation/python.md:85]:
```suggestion
[Configure the layers][1] for your Lambda function using the ARN in the following format:
```

File: content/en/serverless/installation/python.md [hunk 9]
```diff
@@ -90,104 +94,65 @@ For example:
...
 
-### Subscribe the Forwarder to Log Groups
+Or add `datadog-lambda` to your project's `requirements.txt`. See the [latest release][3].
 
-You need the Datadog Forwarder to subscribe to each of your function’s log groups to send traces and enhanced metrics to Datadog.
+### Configure the Function
 
-You can quickly verify that you’ve installed the Datadog Forwarder on the [AWS Forwarder page][1] For more information on the Forwarder ARN, or to install the forwarder see the [official CloudFormation documentation][1]. Make sure the Datadog Forwarder is in the same AWS region as the Lambda functions you are monitoring.
+1. Set your function's handler to `datadog_lambda.handler.handler`.
+2. Set the environment variable `DD_LAMBDA_HANDLER` to your original handler, e.g., `myfunc.handler`.
```
Comment (apigirl) [inline content/en/serverless/installation/python.md:110]:
```suggestion
2. Set the environment variable `DD_LAMBDA_HANDLER` to your original handler, for example, `myfunc.handler`.
```

File: content/en/serverless/installation/python.md [hunk 10]
```diff
@@ -90,104 +94,65 @@ For example:
...
+3. Set environment variable `DD_TRACE_ENABLED` to `true`.
```
Comment (apigirl) [inline content/en/serverless/installation/python.md:111]:
```suggestion
3. Set the environment variable `DD_TRACE_ENABLED` to `true`.
```

File: content/en/serverless/installation/python.md [hunk 11]
```diff
@@ -90,104 +94,65 @@ For example:
...
+4. Set environment variable `DD_FLUSH_TO_LOG` to `true`.
```
Comment (apigirl) [inline content/en/serverless/installation/python.md:112]:
```suggestion
4. Set the environment variable `DD_FLUSH_TO_LOG` to `true`.
```

File: content/en/serverless/installation/python.md [hunk 12]
```diff
@@ -90,104 +94,65 @@ For example:
...
+5. Optionally add a `service` and `env` tag with appropriate values to your function.
 
-1. To start, navigate to your AWS Dashboard for the Datadog Forwarder. Then, manually add a function trigger.
-2. Configure the trigger to be linked to your function’s CloudWatch Log Group, add a filter name (but feel free to leave the filter empty) and add the trigger.
+### Subscribe the Datadog Forwarder to the Log Groups
 
-The Datadog Forwarder will now send enhanced metrics and traces from your function to Datadog.
+You need to subscribe the Datadog Forwarder Lambda function to each of your function’s log groups, in order to send metrics, traces and logs to Datadog.
```
Comment (apigirl) [inline content/en/serverless/installation/python.md:117]:
```suggestion
You need to subscribe the Datadog Forwarder Lambda function to each of your function’s log groups, to send metrics, traces and logs to Datadog.
```

File: content/en/serverless/installation/python.md [hunk 13]
```diff
@@ -90,104 +94,65 @@ For example:
...
 
+1. [Install the Datadog Forwarder if you haven't][4].
```
Comment (apigirl) [inline content/en/serverless/installation/python.md:119]:
```suggestion
1. [Install the Datadog Forwarder][4] if you haven't.
```

File: content/en/serverless/installation/python.md [hunk 14]
```diff
@@ -90,104 +94,65 @@ For example:
...
+2. [Ensure the parameter DdFetchLambdaTags is set to `true`][5].
```
Comment (apigirl) [inline content/en/serverless/installation/python.md:120]:
```suggestion
2. Set the parameter [`DdFetchLambdaTags`][5] to `true`.
```

File: content/en/serverless/installation/python.md [hunk 15]
```diff
@@ -90,104 +94,65 @@ For example:
...
+[2]: https://github.com/DataDog/datadog-lambda-layer-python/releases
+[3]: https://pypi.org/project/datadog-lambda/
+[4]: https://docs.datadoghq.com/serverless/troubleshooting/installing_the_forwarder
+[5]: https://docs.datadoghq.com/serverless/troubleshooting/installing_the_forwarder/#ddfetchlambdatags
+[6]: https://docs.datadoghq.com/integrations/amazon_web_services/?tab=automaticcloudformation#send-aws-service-logs-to-datadog
+{{% /tab %}} 
+{{< /tabs >}}
 
-### Instrument your code
+## Quick Start
```
Comment (apigirl) [inline content/en/serverless/installation/python.md:133]:
```suggestion
## Viewing your data
```

File: content/en/serverless/installation/python.md [hunk 16]
```diff
@@ -90,104 +94,65 @@ For example:
...
 
-Instrument your functions to ingest traces into Datadog:
+After you have configured your function following the steps above, you can view the enhanced metrics, logs and traces on the [Serverless page][2]. If you would like to submit a custom metric or manually instrument a function, see the sample code below.
```
Comment (apigirl) [inline content/en/serverless/installation/python.md:135]:
```suggestion
After you have configured your function following the steps above, you can view the enhanced metrics, logs and traces on the [Serverless page][2]. If you would like to submit a custom metric or manually instrument a function, see the sample code below:
```

File: content/en/serverless/installation/ruby.md [hunk 17]
```diff
@@ -10,16 +10,19 @@ further_reading:
...
 ### Install the Datadog Lambda Library
 
-The Datadog Lambda Library can be imported as a layer. Its ARN includes a region, language runtime, and version. Construct yours in the following format:
+The Datadog Lambda Library can be imported as a layer or a gem.
+
+The minor version of the `datadog-lambda` gem always matches the layer version. E.g., datadog-lambda v0.5.0 matches the content of layer version 5.
+
+#### Using the Layer
+
+[Configure the layers][2] for your Lambda function using the ARN in the following format.
```
Comment (apigirl) [inline content/en/serverless/installation/ruby.md:25]:
```suggestion
[Configure the layers][2] for your Lambda function using the ARN in the following format:
```

File: content/en/serverless/installation/ruby.md [hunk 18]
```diff
@@ -28,85 +31,70 @@ arn:aws:lambda:<AWS_REGION>:464622532012:layer:Datadog-<RUNTIME>:<VERSION>
...
-    });
-});
+gem 'datadog-lambda'
+gem 'ddtrace'
 ```
 
-### Subscribe the Forwarder to log groups
-
-You need the Datadog Forwarder to subscribe to each of your function’s log groups to send traces and enhanced metrics to Datadog.
+Keep in mind that `ddtrace` uses native extensions, which must be compiled for Amazon Linux before being packaged and uploaded to Lambda. For this reason we recommend using the layer.
```
Comment (apigirl) [inline content/en/serverless/installation/ruby.md:48]:
```suggestion
Keep in mind that `ddtrace` uses native extensions, which must be compiled for Amazon Linux before being packaged and uploaded to Lambda. For this reason, Datadog recommends using the layer.
```

File: content/en/serverless/installation/ruby.md [hunk 19]
```diff
@@ -28,85 +31,70 @@ arn:aws:lambda:<AWS_REGION>:464622532012:layer:Datadog-<RUNTIME>:<VERSION>
...
 
-1. To start, navigate to your AWS Dashboard for the Datadog Forwarder. Then, manually add a function trigger.
-2. Configure the trigger to be linked to your function’s CloudWatch Log Group, add a filter name (but feel free to leave the filter empty) and add the trigger.
+### Subscribe the Datadog Forwarder to the Log Groups
 
-The Datadog Forwarder will now send enhanced metrics and traces from your function to Datadog.
+You need to subscribe the Datadog Forwarder Lambda function to each of your function’s log groups, in order to send metrics, traces and logs to Datadog.
```
Comment (apigirl) [inline content/en/serverless/installation/ruby.md:52]:
```suggestion
You need to subscribe the Datadog Forwarder Lambda function to each of your function’s log groups o send metrics, traces, and logs to Datadog.
```

File: content/en/serverless/installation/ruby.md [hunk 20]
```diff
@@ -28,85 +31,70 @@ arn:aws:lambda:<AWS_REGION>:464622532012:layer:Datadog-<RUNTIME>:<VERSION>
...
-
-You can install the Datadog Lambda Library locally by adding the following lines to your Gemfile.
-
-```
-gem 'datadog-lambda'
-gem 'ddtrace'
-```
-
-Then, using the AWS Console or the AWS CLI, add a new `DD_FLUSH_TO_LOG` environment variable set to `true`. This step needs to be repeated for every function you wish to trace.
+After you have configured your function following the steps above, you can view the enhanced metrics, logs and traces on the [Serverless page][8]. If you would like to submit a custom metric or manually instrument a function, see the sample code below.
```
Comment (apigirl) [inline content/en/serverless/installation/ruby.md:60]:
```suggestion
After you have configured your function following the steps above, you can view the enhanced metrics, logs and traces on the [Serverless page][8]. If you would like to submit a custom metric or manually instrument a function, see the sample code below:
```

File: content/en/serverless/installation/dotnet.md [hunk 21]
```diff
@@ -23,32 +23,34 @@ After you have [installed the AWS integration][1], use .NET to instrument your a
 
 ## Configuration
 
-### Enable AWS X-Ray
+### Configure the Function
 
-Navigate to the Lambda function in the AWS console you want to instrument. In the debugging and error handling section, check the box to enable AWS X-Ray.
+1. Enable [AWS X-Ray active tracing][2] for your Lambda function.
```
Comment (Hesperide) [inline content/en/serverless/installation/dotnet.md:28]: @apigirl We'll have this link to our public docs in the very near future.

File: content/en/serverless/installation/dotnet.md [hunk 22]
```diff
@@ -23,32 +23,34 @@ After you have [installed the AWS integration][1], use .NET to instrument your a
...
-### Install the AWS X-Ray client library
+You need to subscribe the Datadog Forwarder Lambda function to each of your function’s log groups, in order to send metrics, traces and logs to Datadog.
 
-Install the [AWS X-Ray client library][2].
+1. [Install the Datadog Forwarder if you haven't][3].
+2. [Ensure the option DdFetchLambdaTags is enabled][4].
+3. [Subscribe the Datadog Forwarder to your function's log groups][5].
 
-### Subscribe the Forwarder to log groups
+## Quick Start
```
Comment (Hesperide) [inline content/en/serverless/installation/dotnet.md:38]: @apigirl This is a V1 for what we'll put in "quick start" - let's chat about what we can put here later this week.

File: content/en/serverless/installation/nodejs.md (hunk 5 above)
Comment (Hesperide) [reply content/en/serverless/installation/nodejs.md:141]: Let's keep it as quick start for now, but we should revisit this section and chat about what we want it to look like longer term @apigirl.

File: content/en/serverless/installation/python.md [hunk 23]
```diff
@@ -10,40 +10,38 @@ further_reading:
...
-            enableDDTracing: true
-            flushMetricsToLogs: true
-            # The Datadog Forwarder ARN goes here.
-            forwarder:
+      datadog:
+        flushMetricsToLogs: true
+        forwarder: # The Datadog Forwarder ARN goes here.
     ```
-    For more information on the Forwarder ARN, or to install the forwarder see the [official CloudFormation documentation][2].
+    For more information on the Forwarder ARN, or to install the forwarder see the [forwarder documentation][2].
```
Comment (Hesperide) [inline content/en/serverless/installation/python.md:40]:
```suggestion
    For more information on the Forwarder ARN, or to install the forwarder see the [Forwarder documentation][2].
```

File: content/en/serverless/installation/java.md [hunk 24]
```diff
@@ -52,51 +52,64 @@ Include the following dependency in your `pom.xml`:
...
-```
+You need to subscribe the Datadog Forwarder Lambda function to each of your function’s log groups, in order to send metrics, traces and logs to Datadog.
 
-### Subscribe the Forwarder to log groups
+1. [Install the Datadog Forwarder if you haven't][4].
+2. [Ensure the option DdFetchLambdaTags is enabled][5].
+3. [Subscribe the Datadog Forwarder to your function's log groups][6].
 
-You need the Datadog Forwarder to subscribe to each of your function’s log groups to send traces and enhanced metrics to Datadog.
+## Quick Start
```
Comment (Hesperide) [inline content/en/serverless/installation/java.md:78]:
```suggestion
## Explore Datadog Serverless Monitoring
```

File: content/en/serverless/installation/nodejs.md (hunk 5 above)
Comment (Hesperide) [reply content/en/serverless/installation/nodejs.md:141]: For now, let's call this "Explore Datadog Serverless Monitoring" in line with what APM and other sections are doing.

File: content/en/serverless/installation/nodejs.md [hunk 25]
```diff
@@ -159,46 +110,62 @@ npm install datadog-lambda-js
...
+[2]: https://github.com/DataDog/datadog-lambda-layer-js/releases
+[3]: https://www.npmjs.com/package/datadog-lambda-js
+[4]: https://docs.datadoghq.com/serverless/troubleshooting/installing_the_forwarder
+[5]: https://docs.datadoghq.com/serverless/troubleshooting/installing_the_forwarder/#ddfetchlambdatags
+[6]: https://docs.datadoghq.com/integrations/amazon_web_services/?tab=automaticcloudformation#send-aws-service-logs-to-datadog
+{{% /tab %}}
+{{< /tabs >}}
 
-You can quickly verify that you’ve installed the Datadog Forwarder on the [AWS Forwarder page][1] For more information on the Forwarder ARN, or to install the forwarder see the [official CloudFormation documentation][1]. Make sure the Datadog Forwarder is in the same AWS region as the Lambda functions you are monitoring.
+## Quick Start
```
Comment (Hesperide) [inline content/en/serverless/installation/nodejs.md:141]:
```suggestion
## Explore Datadog Serverless Monitoring
```

File: content/en/serverless/installation/python.md [hunk 26]
```diff
@@ -90,104 +94,65 @@ For example:
...
+[2]: https://github.com/DataDog/datadog-lambda-layer-python/releases
+[3]: https://pypi.org/project/datadog-lambda/
+[4]: https://docs.datadoghq.com/serverless/troubleshooting/installing_the_forwarder
+[5]: https://docs.datadoghq.com/serverless/troubleshooting/installing_the_forwarder/#ddfetchlambdatags
+[6]: https://docs.datadoghq.com/integrations/amazon_web_services/?tab=automaticcloudformation#send-aws-service-logs-to-datadog
+{{% /tab %}} 
+{{< /tabs >}}
 
-### Instrument your code
+## Quick Start
```
Comment (Hesperide) [inline content/en/serverless/installation/python.md:133]:
```suggestion
## Explore Datadog Serverless Monitoring
```

File: content/en/serverless/installation/ruby.md [hunk 27]
```diff
@@ -28,85 +31,70 @@ arn:aws:lambda:<AWS_REGION>:464622532012:layer:Datadog-<RUNTIME>:<VERSION>
...
-1. To start, navigate to your AWS Dashboard for the Datadog Forwarder. Then, manually add a function trigger.
-2. Configure the trigger to be linked to your function’s CloudWatch Log Group, add a filter name (but feel free to leave the filter empty) and add the trigger.
+You need to subscribe the Datadog Forwarder Lambda function to each of your function’s log groups, in order to send metrics, traces and logs to Datadog.
 
-The Datadog Forwarder will now send enhanced metrics and traces from your function to Datadog.
+1. [Install the Datadog Forwarder if you haven't][5].
+2. [Ensure the option DdFetchLambdaTags is enabled][6].
+3. [Subscribe the Datadog Forwarder to your function's log groups][7].
 
+## Quick Start
```
Comment (Hesperide) [inline content/en/serverless/installation/ruby.md:58]:
```suggestion
## Explore Datadog Serverless Monitoring
```

File: content/en/serverless/installation/go.md [hunk 28]
```diff
@@ -31,50 +31,58 @@ You can install the [Datadog Lambda Library][2] locally by running the following
...
+### Subscribe the Datadog Forwarder to the Log Groups
 
-```
+You need to subscribe the Datadog Forwarder Lambda function to each of your function’s log groups, in order to send metrics, traces and logs to Datadog.
+
+1. [Install the Datadog Forwarder if you haven't][4].
+2. [Ensure the option DdFetchLambdaTags is enabled][5].
+3. [Subscribe the Datadog Forwarder to your function's log groups][6].
+
+## Quick Start
```
Comment (Hesperide) [inline content/en/serverless/installation/go.md:47]:
```suggestion
## Explore Datadog Serverless Monitoring
```

File: content/en/serverless/installation/dotnet.md (hunk 22 above)
Comment (Hesperide) [inline content/en/serverless/installation/dotnet.md:38]:
```suggestion
## Explore Datadog Serverless Monitoring
```

File: content/en/serverless/installation/ruby.md [hunk 29]
```diff
@@ -28,85 +31,70 @@ arn:aws:lambda:<AWS_REGION>:464622532012:layer:Datadog-<RUNTIME>:<VERSION>
...
 
-### Subscribe the Forwarder to log groups
+Keep in mind that `ddtrace` uses native extensions, which must be compiled for Amazon Linux before being packaged and uploaded to Lambda. For this reason we recommend using the layer.
 
-You need the Datadog Forwarder to subscribe to each of your function’s log groups to send traces and enhanced metrics to Datadog.
+### Subscribe the Datadog Forwarder to the Log Groups
 
-1. To start, navigate to your AWS Dashboard for the Datadog Forwarder. Then, manually add a function trigger.
-2. Configure the trigger to be linked to your function’s CloudWatch Log Group, add a filter name (but feel free to leave the filter empty) and add the trigger.
+You need to subscribe the Datadog Forwarder Lambda function to each of your function’s log groups, in order to send metrics, traces and logs to Datadog.
```
Comment (Hesperide) [inline content/en/serverless/installation/ruby.md:52]:
```suggestion
You need to subscribe the Datadog Forwarder Lambda function to each of your function’s log groups to send metrics, traces and logs to Datadog.
```

File: content/en/serverless/installation/dotnet.md [hunk 30]
```diff
@@ -23,32 +23,34 @@ After you have [installed the AWS integration][1], use .NET to instrument your a
...
-Install the [AWS X-Ray client library][2].
+1. [Install the Datadog Forwarder if you haven't][3].
+2. [Ensure the option DdFetchLambdaTags is enabled][4].
+3. [Subscribe the Datadog Forwarder to your function's log groups][5].
 
-### Subscribe the Forwarder to log groups
+## Explore Datadog Serverless Monitoring
 
-You need the Datadog Forwarder to subscribe to each of your function’s log groups to send custom metrics and logs to Datadog.
+After you have configured your function following the steps above, you should be able to view metrics, logs and traces on the [Serverless page][6]. If you need to submit a custom metric, refer to the sample code below.
```
Comment (Hesperide) [inline content/en/serverless/installation/dotnet.md:40]:
```suggestion
After you have configured your function following the steps above, you should be able to view metrics, logs and traces on the [Serverless page][6]. If you want to submit a custom metric, refer to the sample code below:
```

File: content/en/serverless/installation/dotnet.md [hunk 31]
```diff
@@ -23,32 +23,34 @@ After you have [installed the AWS integration][1], use .NET to instrument your a
...
+### Configure the Function
 
-Navigate to the Lambda function in the AWS console you want to instrument. In the debugging and error handling section, check the box to enable AWS X-Ray.
+1. Enable [AWS X-Ray active tracing][2] for your Lambda function.
 
-**Note**: If you’re using a Customer Master Key to encrypt traces, add the `kms:Decrypt` method to your IAM policy where the Resource is the Customer Master Key used for X-Ray.
+### Subscribe the Datadog Forwarder to the Log Groups
 
-### Install the AWS X-Ray client library
+You need to subscribe the Datadog Forwarder Lambda function to each of your function’s log groups, in order to send metrics, traces and logs to Datadog.
```
Comment (Hesperide) [inline content/en/serverless/installation/dotnet.md:32]:
```suggestion
You need to subscribe the Datadog Forwarder Lambda function to each of your function’s log groups to send metrics, traces, and logs to Datadog.
```

File: content/en/serverless/installation/java.md [hunk 32]
```diff
@@ -52,51 +52,64 @@ Include the following dependency in your `pom.xml`:
...
-### Subscribe the Forwarder to log groups
+1. [Install the Datadog Forwarder if you haven't][4].
+2. [Ensure the option DdFetchLambdaTags is enabled][5].
+3. [Subscribe the Datadog Forwarder to your function's log groups][6].
 
-You need the Datadog Forwarder to subscribe to each of your function’s log groups to send traces and enhanced metrics to Datadog.
+## Explore Datadog Serverless Monitoring
 
-You can quickly verify that you’ve installed the Datadog Forwarder by using the [AWS console][2]. If you have not yet installed the Forwarder, you can follow the installation instructions here. Make sure the Datadog Forwarder is in the same AWS region as the Lambda functions you are monitoring.
+After you have configured your function following the steps above, check out the sample code below to get tarted, and view metrics, logs and traces on the [Serverless page][7].
```
Comment (Hesperide) [inline content/en/serverless/installation/java.md:80]:
```suggestion
After you have configured your function following the steps above, check out the sample code below to get started, and view metrics, logs and traces on the [Serverless page][7].
```

File: content/en/serverless/installation/go.md [hunk 33]
```diff
@@ -31,50 +31,58 @@ You can install the [Datadog Lambda Library][2] locally by running the following
...
-```
+You need to subscribe the Datadog Forwarder Lambda function to each of your function’s log groups, in order to send metrics, traces and logs to Datadog.
+
+1. [Install the Datadog Forwarder if you haven't][4].
+2. [Ensure the option DdFetchLambdaTags is enabled][5].
+3. [Subscribe the Datadog Forwarder to your function's log groups][6].
+
+## Explore Datadog Serverless Monitoring
+
+After you have configured your function following the steps above, check out the sample code below to get tarted, and view metrics, logs and traces on the [Serverless page][7].
```
Comment (Hesperide) [inline content/en/serverless/installation/go.md:49]:
```suggestion
After you have configured your function following the steps above, check out the sample code below to get started, and view metrics, logs and traces on the [Serverless page][7].
```

Comment (Hesperide) [inline content/en/serverless/installation/go.md:49]:
```suggestion
After you have configured your function following the steps above, you should be able to view metrics, logs and traces on the [Serverless page][7]. If you need to submit a custom metric, refer to the sample code below:
```

File: content/en/serverless/installation/java.md (hunk 32 above)
Comment (Hesperide) [inline content/en/serverless/installation/java.md:80]:
```suggestion
After you have configured your function following the steps above, you should be able to view metrics, logs and traces on the [Serverless page][7]. If you need to submit a custom metric, refer to the sample code below.
```

File: content/en/serverless/installation/nodejs.md [hunk 34]
```diff
@@ -159,46 +110,62 @@ npm install datadog-lambda-js
...
+[4]: https://docs.datadoghq.com/serverless/troubleshooting/installing_the_forwarder
+[5]: https://docs.datadoghq.com/serverless/troubleshooting/installing_the_forwarder/#ddfetchlambdatags
+[6]: https://docs.datadoghq.com/integrations/amazon_web_services/?tab=automaticcloudformation#send-aws-service-logs-to-datadog
+{{% /tab %}}
+{{< /tabs >}}
 
-You can quickly verify that you’ve installed the Datadog Forwarder on the [AWS Forwarder page][1] For more information on the Forwarder ARN, or to install the forwarder see the [official CloudFormation documentation][1]. Make sure the Datadog Forwarder is in the same AWS region as the Lambda functions you are monitoring.
+## Explore Datadog Serverless Monitoring
 
+After you have configured your function following the steps above, you can view metrics, logs and traces on the [Serverless page][2]. If you would like to submit a custom metric or manually instrument a function, see the sample code below.
```
Comment (Hesperide) [inline content/en/serverless/installation/nodejs.md:143]:
```suggestion
After you have configured your function following the steps above, you can view metrics, logs and traces on the [Serverless page][2]. If you would like to submit a custom metric or manually instrument a function, see the sample code below:
```

File: content/en/serverless/installation/python.md [hunk 35]
```diff
@@ -90,104 +94,65 @@ For example:
...
+[5]: https://docs.datadoghq.com/serverless/troubleshooting/installing_the_forwarder/#ddfetchlambdatags
+[6]: https://docs.datadoghq.com/integrations/amazon_web_services/?tab=automaticcloudformation#send-aws-service-logs-to-datadog
+{{% /tab %}} 
+{{< /tabs >}}
 
-### Instrument your code
+## Explore Datadog Serverless Monitoring
 
-Instrument your functions to ingest traces into Datadog:
+After you have configured your function following the steps above, you can view metrics, logs and traces on the [Serverless page][2]. If you would like to submit a custom metric or manually instrument a function, see the sample code below.
```
Comment (Hesperide) [inline content/en/serverless/installation/python.md:135]:
```suggestion
After you have configured your function following the steps above, you can view metrics, logs and traces on the [Serverless page][2]. If you would like to submit a custom metric or manually instrument a function, see the sample code below:
```

File: content/en/serverless/installation/ruby.md [hunk 36]
```diff
@@ -28,85 +31,70 @@ arn:aws:lambda:<AWS_REGION>:464622532012:layer:Datadog-<RUNTIME>:<VERSION>
...
+1. [Install the Datadog Forwarder if you haven't][5].
+2. [Ensure the option DdFetchLambdaTags is enabled][6].
+3. [Subscribe the Datadog Forwarder to your function's log groups][7].
 
+## Explore Datadog Serverless Monitoring
 
-[1]: https://github.com/DataDog/datadog-lambda-layer-rb/releases
-{{% /tab %}}
-{{% tab "Other" %}}
+After you have configured your function following the steps above, check out the sample code below to get started, and view metrics, logs and traces on the [Serverless page][8].
```
Comment (Hesperide) [inline content/en/serverless/installation/ruby.md:60]:
```suggestion
After you have configured your function following the steps above, you should be able to view metrics, logs and traces on the [Serverless page][8]. If you need to submit a custom metric, refer to the sample code below:
```

File: content/en/serverless/installation/ruby.md (hunk 20 above)
Comment (Hesperide) [reply content/en/serverless/installation/ruby.md]: fixed

File: content/en/serverless/installation/ruby.md (hunk 19 above)
Comment (Hesperide) [reply content/en/serverless/installation/ruby.md]: fixed

File: content/en/serverless/installation/python.md (hunk 16 above)
Comment (Hesperide) [reply content/en/serverless/installation/python.md]: fixed

**Pull request merged**